content/webapp/utils/iiif/v3/index.ts

```diff
@@ -532,3 +534,36 @@ export function getCollectionManifests(manifest: Manifest): Canvas[] {
     .flat();
   return [...firstLevelManifests, ...collectionManifests] as Canvas[];
 }
+
+type CustomSpecificationBehaviors = SpecificationBehaviors | 'placeholder';
+// Whether something is born digital or not is determined at the canvas level within a iiifManifest
+// It is therefore possible to have a iiifManifest that contains:
+// - only born digital items
+// - no born digital items
+// - a mix of the two.
+// We need to know which we have to determine the required UI.
+export function getBornDigitalStatus(manifest: Manifest): BornDigitalStatus {
+  const hasBornDigital = manifest?.items.some(canvas => {
+    const behavior = canvas?.behavior as
+      | CustomSpecificationBehaviors[]
+      | undefined;
+    return behavior?.includes('placeholder') || false;
+  });
+
+  const hasNonBornDigital = manifest?.items.some(canvas => {
+    const behavior = canvas?.behavior as
+      | CustomSpecificationBehaviors[]
+      | undefined;
+    return !behavior?.includes('placeholder') || false;
+  });
+
+  if (hasBornDigital && !hasNonBornDigital) {
+    return 'allBornDigital';
+  } else if (!hasBornDigital) {
+    return 'noBornDigital';
+  } else if (hasBornDigital && hasNonBornDigital) {
+    return 'mixedBornDigital';
+  } else {
+    return 'noBornDigital';
+  }
```

Comment:
Is this the same behaviour? I think we're ignoring the case where `hasBornDigital` & `hasNonBornDigital` are both false and presuming `noBornDigital` which is probably fine?

```suggestion
  if (hasBornDigital && !hasNonBornDigital) {
    return 'allBornDigital';
  } else if (hasBornDigital && hasNonBornDigital) {
    return 'mixedBornDigital';
  } else {
    return 'noBornDigital';
  }
```